deployment/ccip/deploy_home_chain.go

```diff
@@ -100,8 +118,31 @@ func DeployCapReg(lggr logger.Logger, ab deployment.AddressBook, chain deploymen
 		lggr.Errorw("Failed to deploy capreg", "err", err)
 		return nil, err
 	}
+	return capReg, nil
+}
 
-	lggr.Infow("deployed capreg", "addr", capReg.Address)
+func DeployHomeChain(
+	lggr logger.Logger,
+	e deployment.Environment,
+	chain deployment.Chain,
+	rmnHomeStatic rmn_home.RMNHomeStaticConfig,
+	rmnHomeDynamic rmn_home.RMNHomeDynamicConfig,
+	nodeOps []capabilities_registry.CapabilitiesRegistryNodeOperator,
+	nodeP2PIDsPerNodeOpAdmin map[string][][32]byte,
+) (*ContractDeploy[*capabilities_registry.CapabilitiesRegistry], error) {
+	ab := e.ExistingAddresses
```

Comment:
@connorwstein does this look correct? Instead of sending addressbook, I am loading it from `e.ExistingAddresse`s, in this way after the deployment, `e.ExistingAddresses` gets updated with newly deployed ones.